lib/api/index.js

```diff
@@ -261,15 +261,15 @@ app.post('/jira/:clientKey/uninstall', bodyParser, async (request, response) =>
 
 app.post('/jira/:installationId/verify', bodyParser, async (req, response) => {
   const { installationId } = req.params;
-  const installation = await Installation.findById(installationId);
+  const installation = await Installation.findByPk(installationId);
 
   const respondWith = function (message) {
     const data = {
       message,
       installation: { enabled: installation.enabled, id: installation.id, jiraHost: installation.jiraHost },
     };
 
-    return response.send(JSON.stringify(data));
+    return response.json(data);
```

Comment:
`.json` properly sets the content-type and handles the serialization!

Reply:
TIL!